docs/configuration.rst

```diff
@@ -50,15 +50,15 @@ Network
 +------------------------------------------+------------------------------------------------------------------------------------+-------------------+------------------+---------------+
 | Option                                   | Description                                                                        | Type              | Default          | Client/Server |
 +==========================================+====================================================================================+===================+==================+===============+
-| hadoop.cache.data.bookkeeper.port        | The port on which the BookKeeper server is listening.                              | integer           | 8899             | C & S         |
+| rubix.network.bookkeeper.server.port       | The port on which the BookKeeper server is listening.                              | integer           | 8899             | C & S         |
```

Comment:
Fix spacing to align table, otherwise it will not render correctly
(This can be checked with the Rich Diff view for this file)